src/utils/utils/dropdownEnum.ts

```diff
@@ -0,0 +1,18 @@
+import { ObjectEnum } from '@kubevirt-utils/utils/ObjectEnum';
```

Comment:
@pcbailey I feel like using `ObjectEnum` is a pain from the past that we should try to avoid at all cost.
I do understand that you have portions of your feature relying on it, so IMO its ok to use it for porting features like the Virtualization dashboard. But I strongly recommend to move over to plain objects and hooks when implementing new features from now on. cc @avivtur @metalice @yaacov